src/reactapp/src/components/CheckoutForm/CheckoutForm.jsx

```diff
@@ -63,7 +72,7 @@ function CheckoutForm() {
   }
 
   return (
-    <CheckoutFormWrapper initialData={initialData}>
```

Comment:
You're removing the `CheckoutFormWrapper` here, but you don't remove the file itself. Is it still used anywhere else? If not, I'd remove the file from the codebase so we won't be continuing unused code.

Reply:
Good catch @ArjenMiedema . But, i deliberately kept it there if in case someone likes to go with the older approach. But, if you feel it is totally unnecessary, we can remove that from the codebase entirely.

Reply:
Removed the file